Adafruit_Video_Looper/video_looper.py

```diff
@@ -96,6 +97,17 @@ def _load_file_reader(self):
         return importlib.import_module('.' + module, 'Adafruit_Video_Looper') \
             .create_file_reader(self._config)
 
+    def _load_bgimage(self):
```

Comment:
I propose a more robust function:

    def _load_bgimage(self):
        """Load the configured background image and return an instance of it."""
        image = None
        if self._config.has_option('video_looper', 'bgimage'):
            image = self._config.get('video_looper', 'bgimage')
            if not image:
                self._print('Using ' + str(image) + ' as a background')
                image = pygame.image.load(image)
                image = pygame.transform.scale(image, self._size)
        return image